src/import-export/setup.cfg

```diff
@@ -0,0 +1,2 @@
+[bdist_wheel]
```

Comment:
azure-cli-core doesn't support Python 2 any more.

Leaving this would lead to the built wheel named as `import-export-py2-py3-none-any.whl`.

Should Leave this file empty and the wheel name would be `import-export-py3-none-any.whl`

Reply:
updated